src/data/languages/Spanish.json

```diff
@@ -0,0 +1,444 @@
+{
+	"info": {
+		"language": "Español",
+		"language_english": "Spanish",
+		"fallback": ["English"],
+		"maintainer": "MrJako2001",
+		"maintainer_link": "https://anilist.co/user/MrJaco/",
+		"discussion_link": "",
+		"notes": "This is the default language of Automail. This key should have all the translation keys, while other translations don't have to. I'm not a native speaker, but I try to stick to British English here (feel free to britpick!). You can make translation files for other version of English too if you want, name it something like 'English (US)', and put 'English' in the fallback array. You need only include the keys that are different."
+	},
+	"keys": {
+"$loading": "Cargando...",
+"$searching": "Buscando...",
+"$load_more": "Cargar más",
+"$time_now": "Ahora",
+"$time_1second": "Hace un segundo",
+"$time_Msecond": "Hace {0} segundos",
+"$time_1minute": "Hace un minuto",
+"$time_Mminute": "Hace {0} minutos",
+"$time_1hour": "Hace una hora",
+"$time_Mhour": "Hace {0} horas",
+"$time_1day": "Hace un día",
+"$time_Mday": "Hace {0} días",
+"$time_1week": "Hace una semana",
+"$time_Mweek": "Hace {0} semanas",
+"$time_1month": "Hace un mes",
+"$time_Mmonth": "Hace {0} meses",
+"$time_1year": "Hace un año",
+"$time_Myear": "Hace {0} años",
+"$time_medium_Mday": "días",
+"$time_short_second": "s",
+"$time_short_minute": "m",
+"$time_short_hour": "h",
+"$time_short_day": "d",
+"$time_short_week": "w",
+"$time_short_month": "m",
+"$time_short_year": "y",

```

Comment:
This appears to be the English abbreviations. Is that intentional?

Reply:
I thought it more appropriate to leave them as they are.